packages/api/tests/endpoints/test-s3credentials.js

```diff
@@ -0,0 +1,284 @@
+'use strict';
+
+/* eslint-disable lodash/prefer-noop */
+const cryptoRandomString = require('crypto-random-string');
+const test = require('ava');
+const sinon = require('sinon');
+const rewire = require('rewire');
+const randomString = () => cryptoRandomString({ length: 6 });
+
+const randomId = (prefix, separator = '-') =>
+  [prefix, randomString()].filter((x) => x).join(separator);
+
+process.env.EARTHDATA_CLIENT_ID = randomId('edlID');
```

Comment:
The endpoint is not EDL specific.  The tests should not work with `process.env.EARTHDATA_CLIENT_ID`, our code should detect if the correct envs are set